.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        container_image: ["georust/proj-ci:rust-1.49", "georust/proj-ci:rust-1.50"]
+        container_image: ["georust/proj-ci:rust-1.49", "georust/proj-ci:rust-1.53"]
```

Comment:
I figured I'd leave 1.49 in the build matrix as some sort of notional minimum supported rust version.